### ISSUE TITLE
Both MAC and Shiva seem underpowered

### ISSUE THREAD

**Issue by Enduar** [Blaylock1988/VanillaReplacement-GV-Weapons-Pack#27]
While the MAC does appear to do more damage than railguns, the amount seems pretty negligible (it rarely penetrates the first layer of armor- usually destroying two blocks wide/deep at best.

______________________________

Shivas seem to do little to no damage at all, ~~and I have yet to see an observable result from the EMP effect.~~

Shiva's EMP effect works, but seems more inconvenient than it is useful. The ability to shut down internal (still protected by armor) systems has a potentially powerful role, however the way WC does it seems like it's more of a pain in the ass than its worth. It seems to simply turn blocks off as well as a secondary, possibly unrelated effect of also making them appear destroyed. The first effect has a tendency to persist after the EMP effect wears off, meaning your ship ends up with a bunch of randomly toggled-off blocks that is a huge pain in the ass to hunt down. This seems like bad gameplay and is also easily solved by people putting looped timer blocks that exist simply to repeatedly press "on" to negate the EMP's lasting effect.

In other words- good potential, bad execution- probably not worth it in the long run unless we can make the "turn block off" part more intuitive and/or fun for the receiving end.

**Comment by Blaylock1988** [comment]
Shiva EMP radius doubled and duration lasts 10 seconds then disabled blocks should automatically turn back on.

MAC still yet to be addressed
